### PR TITLE
Updated top banner

### PR DESCRIPTION
**Pull request by hartantothio**

### CODE REVIEW

File: assets/sass/main.scss [hunk 1]
```diff
@@ -61,14 +61,13 @@ body {
 
 .nav-company {
     padding: 10px;
+    font-size: 10px;
```
Comment (colinhicks) [inline assets/sass/main.scss:64]:
```suggestion
    font-size: 11px;
```

File: _includes/nav.html [hunk 2]
```diff
@@ -2,8 +2,8 @@
 <noscript><iframe src="https://www.googletagmanager.com/ns.html?id={{ site.google_tag_manager_id }}" height="0"
     width="0" style="display:none;visibility:hidden"></iframe></noscript>
 <!-- End Google Tag Manager (noscript) -->
-<div class="nav-company has-text-centered has-background-black is-uppercase">
-  <a class="has-text-white" target="_blank" href="https://confluent.io">Go to confluent.io</a>
+<div class="nav-company has-text-centered has-background-black has-text-white is-uppercase">
```
Comment (colinhicks) [inline _includes/nav.html:5]:
```suggestion
<div class="nav-company has-text-centered has-background-black has-text-white">
```

File: _includes/nav.html [hunk 3]
```diff
@@ -2,8 +2,8 @@
...
+  Use CL60BLOG to get an additional $60 of free <a class="has-text-white" href="https://www.confluent.io/confluent-cloud" target="_blank">Confluent Cloud</a> - <a class="has-text-white" href="https://www.confluent.io/confluent-cloud-promo-disclaimer" target="_blank">(details)</a>
```
Comment (colinhicks) [inline _includes/nav.html:6]:
```suggestion
  Use promo code CL60BLOG to get an additional $60 of free <a class="has-text-white" href="https://www.confluent.io/confluent-cloud" target="_blank">Confluent Cloud</a> - <a class="has-text-white" href="https://www.confluent.io/confluent-cloud-promo-disclaimer" target="_blank">(details)</a>
```

**Comment by hartantothio** [comment]
@colinhicks - addressed the PR comments. Thank you for the suggestion.

**Comment by bbejeck** [comment]
thanks @hartantothio !

**Pull request merged**